### DISCUSSION TITLE
Mastodon logo icon? :)

### DISCUSSION THREAD

**Discussion by filipesmedeiros** [tailwindlabs/heroicons#900] [Ideas]
I think it has gained enough popularity to grant a space in this library, no?

Thanks for all the work!

**Comment by thcrt** [comment]
Seconded. It would be great to have it for social lists etc.

**Comment by abcmoritz** [comment]
No company / social media icons.

https://twitter.com/steveschoger/status/1589731552514084864

**Reply by filipesmedeiros** [reply]
This makes sense to me! But then I feel an activitypub icon might be nice (actually don't know if there's one already)

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
